pkg/generator/resource_set.go

```
@@ -0,0 +1,63 @@
+package generator
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"slices"
+	"strings"
+
+	"github.com/rhobs/observability-operator/pkg/controllers/util"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)